test/index.js

```diff
@@ -1253,3 +1264,97 @@ describe('def', function() {
   });
 
 });
+
+describe('has long error messages', function() {
+
+  //  hasMethods :: [String] -> a -> Boolean
+  var hasMethods = R.curry(function(names, x) {
+    return x != null &&
+           R.all(function(k) { return typeof x[k] === 'function'; }, names);
+  });
+
+  var Semigroup =
+  $.TypeClass('my-package/Semigroup', hasMethods(['concat']));
+
+  var def = $.create($.env.concat([$.Nullable, Either]));
+
+  var expectedError = function(basename) {
+    var path = __dirname + '/error-messages/' + basename + '.txt';
+    return fs.readFileSync(path).toString();
+  };
```

Comment:
Do you intend to keep this function? In other words, do you think having the error messages in separate text files is cleaner than concatenating string literals in situ?